src/views/Pools/components/Vault/VaultRoiCalculatorModal.tsx

```diff
@@ -42,7 +48,9 @@ export const VaultRoiCalculatorModal = ({
       linkLabel={t('Get %symbol%', { symbol: pool.stakingToken.symbol })}
       earningTokenPrice={pool.earningTokenPrice}
       stakingTokenPrice={pool.stakingTokenPrice}
-      stakingTokenBalance={pool.userData?.stakingTokenBalance}
+      stakingTokenBalance={
+        pool.userData?.stakingTokenBalance ? cakeAsBigNumber.plus(pool.userData?.stakingTokenBalance) : cakeAsBigNumber
```

Comment:
<img width="427" alt="Screen Shot 2022-04-20 at 09 27 39" src="https://user-images.githubusercontent.com/97418926/164230251-1257fd8b-37ba-4b91-9f7c-d5044b3ca0e4.png">

We should disable the button if  `stakingTokenBalance` and `cakeAsBigNumber` is loading.

Reply:
It is related to stakingTokenPrice in roi calculator modal, I will add a check

Reply:
Done